.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import { addDecorator } from '@storybook/react';
+import { addParameters } from '@storybook/client-api';
```

Comment:
is this import needed? (not a blocker, we can remove it)

Reply:
Totally not needed. I removed the import and pushed a new commit.

Thanks again!